client/components/UI/SubNav/Dropdown.jsx

```diff
@@ -78,6 +86,11 @@ export class Dropdown extends React.Component {
             </button>
         );
 
+        const filterValueNormalized = this.state.filterValue.trim().toLowerCase();
+        const filteredItems = filterValueNormalized.length < 1 ? this.props.items : this.props.items.filter(
```

Comment:
This is rendering the first character of the template name as uppercase always. 'kill_template' is becoming 'Kill_template'

Reply:
The first characters are being uppercased, but it's not related to my changes. See [firstCharUpperCase](https://github.com/superdesk/superdesk-planning/blob/master/client/components/UI/SubNav/Dropdown.jsx#L132). I'm happy to remove this function from the dropdown component if you don't mind it affecting your existing labels.

Reply:
firstCharUpperCase was used when the options used were ITEM_TYPE ('event' and 'planning' strings ). We can skip this utility function for user created templates.